### ISSUE TITLE
loss islarge

### ISSUE THREAD

**Issue by rinawhale** [kmaninis/OSVOS-PyTorch#29]
HI, thanks for your sharing， when I run train_online.py, the loss is 403.75 and epoch = 10000,is it normal? Similarly, when I run the code train_parent.py, the network does not converge. Can you give me some suggestions？

**Comment by kmaninis** [comment]
Hi, yes this is normal. The losses are not averaged by tensor size, and that's why the large numbers. Also this is the reason that we use a very small learning rate.

**Comment by rinawhale** [comment]
Thank you very much for your reply.  I still have  two questions：
1. In the case of a large loss, how should I judge that the model has completed network training，since I don't know if the loss of the model has converged. How many epoch should I train? 
2. Will you post code with instance segmentation(combine with the mask rcnn)?
Looking forward to your reply, thank you again！

**Comment by kmaninis** [comment]
Hello,

To answer your questions:
1. For the parent network, we did not optimize on the training iterations. For online training, once the training loss does not decrease, you could stop training. We found out that approx. 2000 iterations are optimal for a sequence of DAVIS.
 
2. We are planning to, but this requires merging with the new Mask RCNN PyTorch version and re-running all experiments, so it will take some time before we release it. Sorry.
